grails-app/controllers/groovyschool/UrlMappings.groovy

```diff
@@ -13,5 +13,6 @@ class UrlMappings {
         "/register"(view:"/register")
         "500"(view:'/error')
         "404"(view:'/notFound')
+        "/login"(view:"/login")
```

Comment:
please, could you move this line up before the "500" line?

Reply:
> please, could you move this line up before the "500" line?

Ok well do so immediately